frontend/apps/reader/modules/readerpaging.lua

```diff
@@ -1029,8 +1029,13 @@ function ReaderPaging:onGotoPageRel(diff)
             goto_end(y)
             goto_end(x)
         elseif new_page > 0 then
+            local previous_page = self.current_page
             self:_gotoPage(new_page)
             goto_end(y, -y_diff)
+            if new_page > previous_page then
```

Comment:
Nothing is necessary for the reverse?

Reply:
> Nothing is necessary for the reverse?

Do some tests yesterday half sleeping and didn't find with the test case the problem happening in reverse order. But I shall retest it.